pino.js

```diff
@@ -44,6 +57,10 @@ function pino (opts, stream) {
     buf: ''
   }
 
+  if (opts.enable === false) {
```

Comment:
I tend to prefer positive logic, I find this easier to reason about but harder to work with in code.

Basically `enable: false` is for convenience, and just an alias over `level: 'silent'`. It's useful because you can do things like `pino({ level: 'info', enable: !!process.env.NO_LOG }),